docs/source/onboarding_resources/computing/index.rst

```diff
@@ -280,22 +280,166 @@ lines from your trainings!
 Your Bash Configuration Files 
 -----------------------------
 
-Bash files contain commands you would enter into a command line, but 
-specifically ones you will use very often. For example, every time you 
-want to open a Jupyter session, the cluster needs certain information 
-and requirements. To find this information, it looks in your Bash files 
-rather than asking you to enter the same information every time. 
+Bash configuration files contain information and commands that are used when 
+interacting with the cluster. When the cluster tries to execute some command 
+line prompts, it will look to your bash configuration files for information 
+or filepaths. 
 
-However, Bash files can be confusing since it is less obvious when the 
-information is being used or what it is used for. Therefore, we have 
-provided a copy-and-paste formatting for information to be added to 
-your Bash files. 
+You should have 2 bash configuration files, your bash profile or :code:`.bash_profile` 
+and your bash rc or :code:`.bashrc`. 
 
-.. todo::
+Once you make edits to either of these files, you will need to log out and then back into 
+the cluster before they will take effect. 
 
-  - Confirm and paste in bash files 
-  - Get Zeb's help in explaining Bash files better 
-  - Moving up since this is needed for a lot of things below 
+bash profile
+~~~~~~~~~~~~
+
+The bash profile is only run when you open the cluster, and so is usually a shorter file. 
```

Comment:
```suggestion
The bash profile is only run when you log in to the cluster, and so is usually a shorter file. 
```
Based on [this reference](https://www.tutorialspoint.com/bashrc-vs-bash-profile-what-is-difference#:~:text=Bash_profile%20is%20another%20configuration%20file,at%20start%20of%20your%20session.) and [this reference](https://apple.stackexchange.com/questions/51036/what-is-the-difference-between-bash-profile-and-bashrc), I think you mean "log in to" when you say "open". Does that sound right?

Reply:
Yeah - makes sense!